### PR TITLE
Publish snapshot

### PR DESCRIPTION
**Pull request by maciej-brzezinski**

### CODE REVIEW

File: .github/workflows/helm-test-workflow.yaml [hunk 1]
```diff
@@ -82,4 +86,14 @@ jobs:
         run: helm upgrade -i "${{env.RELEASE_NAME}}" dist/*.tgz --version "${{needs.build.outputs.version}}" --wait --debug -f deploy-values.yaml -f deploy-values-lite.yaml  --set hermes.enabled=false
       - name: helmTestRelease
         run: ./test.sh "${{env.RELEASE_NAME}}"
-
+  publish-snapshot:
+    name: publish-release
```
Comment (mproch) [inline .github/workflows/helm-test-workflow.yaml:87]: let's name it publish-snapshot :)

File: .github/workflows/helm-test-workflow.yaml [hunk 2]
```diff
@@ -82,4 +86,14 @@ jobs:
...
+    if: (startsWith(github.ref, 'refs/tags/*') != true)
```
Comment (mproch) [inline .github/workflows/helm-test-workflow.yaml]: this should run only on push, not on PR

File: .github/workflows/helm-test-workflow.yaml [hunk 3]
```diff
@@ -82,4 +83,13 @@ jobs:
         run: helm upgrade -i "${{env.RELEASE_NAME}}" dist/*.tgz --version "${{needs.build.outputs.version}}" --wait --debug -f deploy-values.yaml -f deploy-values-lite.yaml  --set hermes.enabled=false
       - name: helmTestRelease
         run: ./test.sh "${{env.RELEASE_NAME}}"
-
+  publish-snapshot:
```
Comment (mproch) [inline .github/workflows/helm-test-workflow.yaml:86]: this should run only on push to main, like here:
https://github.com/TouK/nussknacker/blob/staging/.github/workflows/pr.yml#L554

File: .github/workflows/helm-test-workflow.yaml (hunk 1 above)
Comment (maciej-brzezinski) [reply .github/workflows/helm-test-workflow.yaml]: i see it, fixed it, thx

File: .github/workflows/helm-test-workflow.yaml (hunk 2 above)
Comment (maciej-brzezinski) [reply .github/workflows/helm-test-workflow.yaml]: you mean just this job correct?

Comment (maciej-brzezinski) [reply .github/workflows/helm-test-workflow.yaml]: think it's fixed now

File: .github/workflows/helm-test-workflow.yaml (hunk 3 above)
Comment (maciej-brzezinski) [reply .github/workflows/helm-test-workflow.yaml:86]: changed it, should be ok now

**Review by mproch — APPROVED** [review]

**Pull request merged**